common/lib/xmodule/xmodule/tests/test_lti_unit.py

```diff
@@ -69,8 +71,12 @@ def setUp(self):
         current_user = self.system.service(self.xmodule, 'user').get_current_user()
         self.user_id = current_user.opt_attrs.get(ATTR_KEY_ANONYMOUS_USER_ID)
         self.lti_id = self.xmodule.lti_id
+
+        self.hostname = "edx.org"
+        django.conf.settings.LMS_BASE = self.hostname
```

Comment:
Nit: is there a specific reason why we want to assign it like this instead of putting `@override_settings(LMS_BASE="edx.org")` above `LTIBlockTest`, and then retrieving it with `settings.LMS_BASE` while creating `self.unquoted_resource_link_id`?

Reply:
I have updated this as per your comments.

I was mostly trying to avoid converting a `unittest.TestCase` into a Django TestCase. From the way the code is written, it seems XModule was deliberately trying to avoid any Django dependencies. Any reason, this has changed in the refactor and we are more comfortable depending on Django methods?

Reply:
@tecoholic,
> I was mostly trying to avoid converting a unittest.TestCase into a Django TestCase. From the way the code is written, it seems XModule was deliberately trying to avoid any Django dependencies. Any reason, this has changed in the refactor and we are more comfortable depending on Django methods?

It was, but I believe it's not relevant [anymore](https://github.com/openedx/edx-platform/pull/30450#pullrequestreview-994960792). Especially for this case, as the old LTI XBlock [is deprecated](https://github.com/openedx/edx-platform/blob/702866b22579558c2ba88b56a46ba3959a4b9cf0/common/lib/xmodule/xmodule/lti_module.py#L290).